ofnetAgent.go

```diff
@@ -137,6 +138,7 @@ func NewOfnetAgent(bridgeName string, dpName string, localIp net.IP, rpcPort uin
 	agent.MyPort = rpcPort
 	agent.MyAddr = localIp.String()
 	agent.dpName = dpName
+	agent.arpMode = "proxy"
```

Comment:
Minor fix: Could we have constants for this? For understanding, it's better to have constants list down all the supported modes in one place than check it all over the code.